### PR TITLE
Migrating unique bigNumber(total) controls

### PR DESCRIPTION
**Pull request by rusackas**
### CATEGORY

Choose one

- [ ] Bug Fix
- [ ] Enhancement (new features, refinement)
- [x] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY

Moving unique control(s) from `controls.jsx` to `/explore/controlPanels/BigNumber.js` and `/explore/controlPanels/BigNumberTotal.js`. Adding a file `/explore/controlPanels/Shared_BigNumber.js` that applies to both of these exports in this plugin package. I plan to follow this same approach for controls that are shared among multiple exports from one plugin, e.g. DeckGL or NVD3... just to keep things a _little_ more DRY.

### ADDITIONAL INFORMATION

- [x] Has associated issue: #9187
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Review by villebro — APPROVED** [review]

File: superset-frontend/src/explore/controlPanels/BigNumberTotal.js [hunk 1]
```diff
@@ -17,6 +17,7 @@
  * under the License.
  */
 import { t } from '@superset-ui/translation';
+import { HeaderFontSize, SubheaderFontSize } from './Shared_BigNumber';
```
Comment (rusackas) [inline superset-frontend/src/explore/controlPanels/BigNumberTotal.js:20]: @kristw there are only a couple of shared controls between the two BigNumber viz modules, so I made this file to share them within that viz'package/repo. This is just a small example, but I was planning to follow the same example for the NVD3 and DeckGL modules, and consolidate shared controls that are relevant to only their repos.

When we get to the SUPER commonly used controls, we could follow this pattern again, and move the "global" controls (or whatever we call it) to `superset-ui`.

This also means we can still remove `controlOverrides` sectopms from the plugin control files, and do something like:
```
controlSetRows: [
        [Object.assign({},HeaderFontSize,{title: 'someNewTitle'}],
      ],
```
... if that's simpler to interpret.

File: superset-frontend/src/explore/controlPanels/Shared_BigNumber.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// These are control configurations that are shared ONLY within the BigNumber viz plugin repo.
+import { t } from '@superset-ui/translation';
+
+export const HeaderFontSize = {
```
Comment (villebro) [inline superset-frontend/src/explore/controlPanels/Shared_BigNumber.js:23]: Probably stupid question regarding conventions, but I thought this should be `headerFontSize`?

**Review by villebro — APPROVED** [review]
Just a small question/comment, in general LGTM. I like the approach of sharing commonly used controls somehow if they truly denote the same thing.

File: superset-frontend/src/explore/controlPanels/Shared_BigNumber.js (hunk 2 above)
Comment (rusackas) [reply superset-frontend/src/explore/controlPanels/Shared_BigNumber.js:23]: Not a stupid question at all, you're right. That's what I get for doing this late at night while distracted by the TV.

**Comment by rusackas** [comment]
> Just a small question/comment, in general LGTM. I like the approach of sharing commonly used controls somehow if they truly denote the same thing.

Yeah, I think this makes sense on a per-plugin basis, and will do this for Deck/NVD3 as well (today, I hope!)

**Review by villebro — APPROVED** [review]

**Pull request merged**